lib/love.lua

```diff
@@ -227,10 +235,23 @@ love.physics.newFixture = function(body, shape)
     }
 end
 love.physics.newPolygonShape = function()
-    return 'polygon'
+    return {
+        'polygon',
```

Comment:
What's the reason for the stray string values in these tables?

Reply:
That was already there from either you or Trevor, so I left it there when I added the getType stub

Reply:
No it wasn't. The function used to return a (placeholder) string. Now it is returning the table you created.

Reply:
Are you saying that the placeholder string wasnt necessary, and instead of returning a table with it  , I could just return the getType function? Or return both without a table?

Reply:
Does this resolve your issue? : [3fbb236](https://github.com/codi-hacks/space-race/pull/69/commits/3fbb2368e8c1c486603bb3ccbb4a385173a2b3d5)

Reply:
Deleting exactly the one line of code I highlighted would have cleaned it up, but that works too.